### DISCUSSION TITLE
Do macros work with native GameCube controllers?

### DISCUSSION THREAD

**Discussion by abs3ntdev** [darthcloud/BlueRetro#649] [Q&A]
If I have a GameCube controller plugged into the port can I use the macros on that controller to enter deep sleep for example? Or a wavebird controller.

**Comment by darthcloud** [comment]
no BlueRetro is not monitoring the port

**Reply by abs3ntdev** [reply]
Is that a plan for the future or will it never monitor the port for macros?
